src/ocm/components/clusterDetail/RebootNodeZeroModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Button, Checkbox, Modal, ModalVariant, Text, TextContent } from '@patternfly/react-core';
+
+import { useClusterStatusVarieties } from './ClusterDetailStatusVarieties';
+import { Cluster, KubeconfigDownload, ClusterCredentials } from '../../../common';
+
+import './RebootNodeZeroModal.css';
+
+// TODO(mlibra): Following will be reimplemented based on future decisions in https://issues.redhat.com/browse/AGENT-522
```

Comment:
Here are my thoughts on this part:  
1. This component won't be seen in OCM, TTBOMK there's no feature requiring this modal however, lives as part of the OCM-part of the lib... (but you see were this is going in the general case)
2. The day the mentioned decision behind AGENT-522 happens we'll need to release a new version of the ui-lib.

So it seems like we will need to release versions of the "ocm-lib" because either they are required for ABI and/or OCM. I think we should gather and brainstorm around this solution or rethink what is "shared/common code" before we get ourselves into a maintenance nightmare.

Reply:
We had a meeting today where we agreed on the following steps:
1. Merge this PR as a WIP step.
2. A project redesign is required to ensure parts of the lib can be handed over in the future.
3. Open a long-term epic in order to begin decomposing the lib/s into smaller chunks.
4. Use the mono-repo as the hosting infrastructure for this refactor.